deploy/gk-deploy

```diff
@@ -94,6 +123,26 @@ Options:
               from the GlusterFS daemonset. This allows for multiple GlusterFS
               daemonsets to run in the same cluster. Default is 'glusterfs'.
 
+  --no-block
+              Don't deploy a gluster-block container. Default is to deploy.
+
+  --no-object
+              Don't deploy a gluster-s3 container. Default is to deploy.
+
+  --object-account ACCOUNT, --object-user USER, --object-password PASSWORD
+              Required credentials for deploying the gluster-s3 container. If
+              any of these are missing, object container deployment will be
+              skipped.
+
+  --object-sc STORAGE_CLASS
+              Specify an optional, pre-existing StorageClass for the object
+              container to use to create GlusterFS volumes to host the objects.
+              Default is to create a new StorageClass.
+
+  --object-capacity CAPACITY
+              The total capacity of the GlusterFS volume which will host the
+              objects. Default is '${OBJ_CAPACITY}'.
+
```

Comment:
If the user omits even one of the three auth parameters gluster-s3 can't be deployed and thus it is not deployed by default. The default capacity is 2Gi, as per your example.